### PR TITLE
Change to new APIs and support Shahe Campus

### PR DESCRIPTION
**Pull request by RinChanNOWWW**
# Summary

related issue: #2 

## What did I do

- Change the APIs to new APIs.
- Support 沙河 Campus (But need more tests).

## What I didn't do

- Update docs.
- Update the version number.
- Pass all the unit tests.
- Publish to PyPI.

Feel free to ask me for more detail information!

### CODE REVIEW

**Review by jerrymakesjelly — APPROVED** [review]

**Comment by jerrymakesjelly** [comment]
Thank you! @RinChanNOWWW

File: buptelecmon/main.py [hunk 1]
```diff
@@ -17,26 +17,35 @@ def convert_rat(remaining_hrs):
...
+    ret = 0
+    try:
+        ret = float(v)
+    except:
+        pass
+    return ret
+
+
 # Output formater
 def output(dormitory, data, params=None):
```
Comment (jerrymakesjelly) [inline buptelecmon/main.py]: The meanings of data fields in Shahe Campus may have been changed. We need to differentiate between different campuses.

See the screenshot below.

**Review by jerrymakesjelly — CHANGES_REQUESTED** [review]

File: buptelecmon/main.py (hunk 1 above)
Comment (RinChanNOWWW) [reply buptelecmon/main.py]: Fine.

File: buptelecmon/main.py [hunk 2]
```diff
@@ -48,6 +53,16 @@ def output(dormitory, data, params=None):
             if convert_to_float(data['pTotal']) != 0 else 'Infinite')
     )
 
+def output_shahe(data, params=None):
+    # Shahe Campus's data format is different from Xitucheng Campus
+    print('%s %s - Surplus money: %.2f Yuan\t Total power consumption: %.2f kWh).' %
```
Comment (jerrymakesjelly) [inline buptelecmon/main.py]: Redunant parenthese `)` in `%.2f kWh)`, please remove it 😄

**Review by jerrymakesjelly — CHANGES_REQUESTED** [review]

File: buptelecmon/main.py (hunk 2 above)
Comment (RinChanNOWWW) [reply buptelecmon/main.py]: Sorry, didn't note this.

**Review by jerrymakesjelly — APPROVED** [review]

**Pull request merged**